InnoTopicWebsite/src/app/TopicFriendsShared3/topics-core/print.service.ts

```diff
@@ -7,7 +7,7 @@ import {filter} from "rxjs";
 })
 export class PrintService {
 
-  static isPrint = false // !! JSON.parse(window.localStorage.getItem('isPrint') ?? 'false') // true // hack for now
```

Comment:
@samyakjainnnnn This needs to be `false` because it will get set to true automatically when you are on `/print` route